```
stderr	Caused by: java.lang.ExceptionInInitializerError: Exception java.lang.ExceptionInInitializerError: Cannot unpack liblz4-java: java.io.IOException: No such file or directory [in thread "pool-4-thread-2"]
stderr	\tat net.jpountz.util.Native.load(Native.java:140)
stderr	\tat net.jpountz.lz4.LZ4JNI.<clinit>(LZ4JNI.java:31)
```

--- a/client/go/internal/admin/jvm/configproxy_jvm.go
+++ b/client/go/internal/admin/jvm/configproxy_jvm.go
@@ -6,6 +6,7 @@ package jvm
 import (
 	"strings"
 
+	"github.com/vespa-engine/vespa/client/go/internal/admin/defaults"
 	"github.com/vespa-engine/vespa/client/go/internal/admin/prog"
 )
 
@@ -40,7 +41,7 @@ func (cpc *ConfigProxyJvm) ConfigureOptions(configsources []string, userargs str
 	opts.AddOption("-XX:-OmitStackTraceInFastThrow")
 	opts.AddOption("-XX:ActiveProcessorCount=2")
 	opts.AddOption("-Dproxyconfigsources=" + strings.Join(configsources, ","))
-	opts.AddOption("-Djava.io.tmpdir=${VESPA_HOME}/var/tmp")
+	opts.AddOption("-Djava.io.tmpdir=" + defaults.UnderVespaHome("var/tmp"))
 	if userargs != "" {
 		opts.AddJvmArgsFromString(userargs)
 	}

